Backends/CLX/medium.lisp

```diff
@@ -384,9 +399,10 @@ translated, so they begin at different position than [0,0])."))
                   (if (< (ldb (byte 8 0) elt) #x80)
                       (setf (aref mdata y x) 0)
                       (setf (aref mdata y x) 1)))))
-      (unless (or (>= width 2048) (>= height 2048)) ;### CLX bug
-        (xlib:put-image mm mm-gc mm-image :src-x 0 :src-y 0 :x 0 :y 0
-                        :width width :height height :bitmap-p nil))
+      (if (or (>= width 2048) (>= height 2048)) ;### CLX bug
```

Comment:
Maybe write this using `(and (< width 2048) (< height 2048))` and flipping the legs? Having the same spelling of the condition makes it a bit easier to read.